src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -57,7 +65,8 @@ function CustomFieldInputMultipleChoice(props) {
     const selectedChoiceIndex = choicesRefs.findIndex(choiceRef => choiceRef === event.target);
     const selectedChoice = visibleChoices[selectedChoiceIndex];
     setExpanded(false);
-    setValue([...value, selectedChoice]);
+    const newValue = [...value, selectedChoice].sort((a, b) => a.id - b.id);
```

Comment:
From a UI perspective, we might not want to sort the selected choices. It might cause "unexpected" behavior since it is not apparent that a choice was actually selected if it gets sorted in a large list. However, I think this is something we can iterate on instead of holding up this PR -- possibly yet another pain point with the custom fields API.

Reply:
Agreed, it kinda sux. Perhaps there's something to be done like only sorting the `value` return, but that does feel like concealing the real problem, which is that the API sorts the selected values instead of maintaining their order.